README.md

```diff
@@ -60,14 +60,16 @@ This project is the desktop version of the portfolio project.
 
 ### Key Features <a name="key-features"></a>
 
-> Added a responsive new homepage, works-section and a contact-section to the portfolio project. 
+> Added a responsive new homepage, works-section and a contact-section to the portfolio project.
+> Employed Flexbox and grid.
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+## 🚀 Live Demo click <a href="https://nasirsethe.github.io/Mobile-Portfolio/">here</a>
 
-> Coming soon.
+> Click <a href="nasirsethe.github.io>here<a> for a Live Preview.
 
```

Comment:
> [OPTIONAL] Well done, @NasirSethe👍! Your website is working flawlessly. Would you be so kind as to move the live link to line 72 rather than including it in the section heading? This will improve the organization and clarity of your readme contents.😊
> 
> ![readme](https://github.com/NasirSethe/Mobile-Portfolio/assets/40009354/7b9d6d28-306c-410b-ae26-43b68b206387)
>

Reply:
@Johnadibe will make the changes thank you for the suggestion  🙏🏾